### PR TITLE
Update: Switch from master to main branch

### PR DESCRIPTION
**Pull request by iokpala**
# Summary | Résumé

Going forward, we are adopting main as our default branch,

This commit updates the workflows YAML files, replacing master with  main branch

### CODE REVIEW

**Comment by jimleroyer** [comment]
Spoke with @iokpala who started on the effort. This is a low hanging fruit task to do in-between tasks, while waiting for new ones.

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ jobs:
         app_id: ${{ secrets.PRBOT_APP_ID }}
         private_key: ${{ secrets.PRBOT_PRIVATE_KEY }}
 
-    - uses: cds-snc/notification-pr-bot@master
+    - uses: cds-snc/notification-pr-bot@main
```
Comment (sastels) [inline .github/workflows/docker.yaml:73]: This is referencing the pr-bot's default branch, which is `master`. ☝️  is a good idea though, just that we'll have to first change pr-bot so it has a `main` branch.

File: .github/workflows/secret.yaml [hunk 2]
```diff
@@ -5,6 +5,6 @@ jobs:
     name: seekret-scanning
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@main
     - name: docker://cdssnc/seekret-github-action
       uses: docker://cdssnc/seekret-github-action
```
Comment (sastels) [inline .github/workflows/secret.yaml:10]: I'm not sure if this "master" refers to the admin repo. 🤔  Wouldn't that mean that in CI we never checked the PR branch, just the default branch? @maxneuvians you set all this up, what's going on here?

**Review by sastels — COMMENTED** [review]
I'm not sure if there's any changes we need before proceeding, just a couple things to work out first.

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (iokpala) [reply .github/workflows/docker.yaml:73]: Good catch.. Cheers Now has a `main`  branch.. https://github.com/cds-snc/notification-pr-bot/tree/main

File: .github/workflows/secret.yaml (hunk 2 above)
Comment (iokpala) [reply .github/workflows/secret.yaml]: Good catch!! Cheers

Comment (sastels) [reply .github/workflows/secret.yaml]: Chatted with Max, the master there does refer to the admin repo. So we could switch to "main" to replicate the behaviour. BUT this is scanning the default branch, NOT the branch in this PR. So we should switch it to
```
- uses: actions/checkout@v2
```
and then we'd be scanning for secrets added in the PR (which is better than not knowing until the PR is merged)

**Review by jimleroyer — APPROVED** [review]

**Comment by sastels** [comment]
Looks good! As mentioned above we should switch the seekret scanner to either be
```
    - uses: actions/checkout@main
```
to keep the original functionality
or 
```
    - uses: actions/checkout@v2
```
to do the scanning on the PR branch rather than the default branch.

**Comment by sastels** [comment]
Note that merging this PR will probably trigger a deploy to staging (since we're now deploying when we merge to main)

**Comment by iokpala** [comment]
@sastels noted

**Pull request merged**